flood_adapt/integrator/interface/hazard_adapter.py

```diff
@@ -0,0 +1,57 @@
+from abc import abstractmethod
+
+from flood_adapt.integrator.interface.model_adapter import IAdapter, ModelData
+from flood_adapt.object_model.hazard.interface.events import IEventModel
+from flood_adapt.object_model.hazard.interface.forcing import IForcing
+from flood_adapt.object_model.hazard.measure.hazard_measure import HazardMeasure
+from flood_adapt.object_model.hazard.physical_projection import PhysicalProjection
+
+
+class HazardData(ModelData):
+    pass
+
+
+class IHazardAdapter(IAdapter):
+    @abstractmethod
+    def set_timing(self, event: IEventModel):
```

Comment:
Well, this sets the timing of the hazard model from the event model. I dont see how this could be moved to the event class since this is essentially a hydormt_sfincs call. 
I do agree that this maybe should be a part of this interface as other models might not need the same eventmodel wrt the timing